includes/admin/class-wp-job-manager-addons-landing-page.php

```diff
@@ -46,7 +46,8 @@ public static function instance() {
 	 * @since $$next-version$$
 	 */
 	public function __construct() {
-		add_action( 'admin_menu', [ $this, 'add_menu_items' ], 12 );
+		add_action( 'admin_menu', [ $this, 'add_applications_menu_item' ], 12 );
+		add_action( 'admin_menu', [ $this, 'add_resumes_menu_item' ], 12 );
```

Comment:
```suggestion
		add_action( 'admin_menu', [ $this, 'add_menu_items' ], 12 );
```
Not a blocker: Instead of having two actions there, could we have just one, calling add_menu_items, then add_menu_items calls both methods ? It might feel a bit clearer and easier to maintain

Reply:
I started out with that, but eliminated that method since it wasn't adding much value, just calling the two other ones each doing their own thing. Seemed a bit better to have two more granular hooks, but I don't have strong opinions about it.

For these cases where the method has to be `public` for hooks but otherwise is not really a public method, we add the `@access private` annotation to indicate that it shouldn't be used directly.